### PR TITLE
trans content/zh/docs/_index.md

### PR DESCRIPTION
**Pull request by chuanhaojin**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

trans content/zh/docs/_index.md

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: content/zh/docs/_index_zh.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+lastmod: 2019-01-29
+menu:
+  docs:
+    parent: welcome
+    weight: 1
+title: 入门指南
+toc: true
+type: docs
+---
+KubeEdge是一个开源系统，将本机容器化应用程序编排功能扩展到边缘主机。
```
Comment (kevin-wangzefeng) [inline content/zh/docs/_index_zh.md:13]:
```suggestion
KubeEdge是一个开源系统，将原生的容器化应用程序编排功能扩展到边缘节点。
```

File: content/zh/docs/_index_zh.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## 获取KubeEdge！
+
+您可以在[这里](https://github.com/kubeedge/kubeedge/releases)找到最新的KubeEdge版本。
+
+在发布期间，我们为主要平台构建tarball，并在kubeedge dockerhub中发布docker映像。
+
+## 部署KubeEdge
+
+查看 [设置文档](./setup/keadm) 。
```
Comment (kevin-wangzefeng) [inline content/zh/docs/_index_zh.md:44]:
```suggestion
查看 [安装文档](./setup/keadm) 。
```

File: content/zh/docs/_index_zh.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## 贡献
+
+贡献很受欢迎！请参见我们的[CONTRIBUTING.md](./community/contribute)获得更多的信息。
```
Comment (kevin-wangzefeng) [inline content/zh/docs/_index_zh.md:48]:
```suggestion
欢迎贡献！请参见我们的[CONTRIBUTING.md](./community/contribute)获得更多的信息。
```

**Comment by kevin-wangzefeng** [comment]
Minor comments, thanks.

**Comment by chuanhaojin** [comment]
> Minor comments, thanks.

Have been updated. Thanks for your attention. :)

**Comment by kevin-wangzefeng** [comment]
/lgtm
/approve

Thanks.

**Pull request merged**